Script/AtomicEditor/ui/frames/ResourceFrame.ts

```diff
@@ -226,6 +226,23 @@ class ResourceFrame extends ScriptWidget {
         }
     }
 
+    handleUserPreferencesChanged() {
```

Comment:
This is way better now with script access to the view, though we really need to refactor the initial version of editors to make it more pluggable... not for this PR!